api/agent/agent.go

```diff
@@ -1302,7 +1247,10 @@ func newHotContainer(ctx context.Context, call *call, cfg *Config) (*container,
 			for _, b := range bufs {
 				bufPool.Put(b)
 			}
-			closer() // XXX(reed): clean up
+			if err = iofs.Close(); err != nil {
+				// Note: This is logged with the context of the container creation
```

Comment:
yea, we could fix this (think you've brought it up). need to plumb the driver.ContainerTask.Close(ctx) method (can add a note, this is parity anyway).

would leave a very loud note here that if anybody decides to shuffle things above, `iofs.Close()` MUST be called or we leak dirs... (or `XXX(reed):` me and i'll get to it this week)

Reply:
Done